### PR TITLE
WIP: Webtest

### PR DESCRIPTION
**Pull request by plutonik-a**
Add Selenium web tests for HSG Apps

### CODE REVIEW

File: webtest/guides/test/util.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var loginForm = browser.element('form#guides-login');
+
+browser.addCommand('login', function() {
+    browser.waitForVisible('form#guides-login');
+    loginForm.click('input[name="user"]');
+    browser.setValue('input[name="user"]', 'style');
+    loginForm.click('input[name="password"]');
+    browser.setValue('input[name="password"]', 'gu1d3');
```
Comment (joewiz) [inline webtest/guides/test/util.js]: Hmm...

**Comment by joewiz** [comment]
@plutonik-a Please find an alternative approach to storing sensitive data in publicly accessible source control.

**Comment by plutonik-a** [comment]
@joewiz Could you please delete the branch webtest for me?  
I will create a new PR with the alternative login solution.
So there is no need to continue with this branch since I have a corrupted history here that could cause problems.

**Comment by joewiz** [comment]
Sure, here goes.

**Pull request closed without merging**